open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/openlineageconnectors/janusconnector/factory/GraphFactory.java

```diff
@@ -48,9 +48,8 @@ public JanusGraph openGraph(String graphDB,ConnectionProperties connectionProper
         JanusGraphFactory.Builder config =janusFactoryBeans.getJanusFactory(graphDB,connectionProperties,graphType);
 
         try {
-
             janusGraph = config.open();
-            return initialize(janusGraph,graphType);
+            return initializeGraph(janusGraph,graphType);
         } catch (Exception e) {
             log.error("{} could not open graph store", e.getMessage());
```

Comment:
this log might be confusing as you don't know what is the exception message.
Maybe:
`log.error("Error opening graph store", e);`